static/js/floating_recipient_bar.js

```diff
@@ -16,8 +16,9 @@ export function first_visible_message(bar) {
     // overlaps the floating recipient bar's space (since you ).
 
     const messages = bar.children(".message_row");
-    const frb_bottom = get_frb_bottom();
-    const frb_top = frb_bottom - 25;
+    const frb = $("#floating_recipient_bar");
+    const frb_top = top_offset(frb);
+    const frb_bottom = frb_top + frb.safeOuterHeight();
```

Comment:
Am I correct that this is removing a `- 25`?  If so, the commit message should explain why we're doing so and why it's correct to do so.

Reply:
In the diff above, actually the `frb_bottom` and `frb_top` line ordering is interchanged.
We're doing the same as : `const frb_bottom = frb_top + 25;`
___

( Also, oops I notice I pushed from master branch. )